lib/install/action/extract.js

```diff
@@ -68,7 +69,9 @@ function stageBundledModule (bundler, child, staging, parentPath, next) {
 
   function moveModule () {
     if (child.fromBundle) {
-      return move(stageFrom, stageTo, iferr(next, updateMovedPackageJson))
+      return mkdirp(path.dirname(stageTo), iferr(next, function () {
```

Comment:
We need to mkdirp so that `.staging/@scope` is created before we call move. A separate bug was masking the error so that we didn't know that it had failed until `finalize` failed.